errors/error.go

```diff
@@ -4,6 +4,7 @@ package errors
 import (
 	"bytes"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
Is it possible that importing this causes issues for people not using the package already?

Reply:
it pulls in the `errors` package as a dependency, where it wasn't already. I wish it wasn't necessary to import it at all and could redefine the `StackTrace()` interface internally.